app/overrides/o_data_controller_override.rb

```diff
@@ -3,21 +3,27 @@
 # Here we re-open odata_server's main controller
 # to add NEMO things like before_action.
 ODataController.class_eval do # rubocop:disable Metrics/BlockLength
-  before_action :refresh_schema
+  NAMESPACE = "NEMO"
```

Comment:
Maybe house this constant in one of the other OData classes? It could be something about class_eval is causing that warning.